q2_alignment/_filter.py

```diff
@@ -79,4 +79,19 @@ def mask(alignment: skbio.TabularMSA, max_gap_frequency: float=0.05,
                                                    min_conservation)
     combined_mask = gap_mask & conservation_mask
     # apply the mask and return the resulting alignment
-    return _apply_mask(alignment, combined_mask)
+    result = _apply_mask(alignment, combined_mask)
+
+    if result.shape[1] == 0:
+        num_input_positions = alignment.shape[1]
+        percent_passed_gap_filter = \
+            (gap_mask.sum() / num_input_positions) * 100.
```

Comment:
Can `num_input_positions` be zero? If so, may want to check input earlier in the method to disallow alignments with no positions.